declarations/WebpackOptions.d.ts

```diff
@@ -0,0 +1,1249 @@
+				 */
+				usedExports?: boolean;
+				/**
+				 * show reasons why optimization bailed out for modules
+				 */
+				optimizationBailout?: boolean;
+		  }
+		| boolean
+		| ("none" | "errors-only" | "minimal" | "normal" | "detailed" | "verbose");
+	/**
+	 * Environment to build for
+	 */
+	target?:
+		| (
+				| "web"
+				| "webworker"
+				| "node"
+				| "async-node"
+				| "node-webkit"
+				| "electron-main"
+				| "electron-renderer")
+		| {
+				[k: string]: any;
+		  };
+	/**
+	 * Enter watch mode, which rebuilds on file change.
+	 */
+	watch?: boolean;
+	/**
+	 * Options for the watcher
+	 */
+	watchOptions?: {
+		/**
+		 * Delay the rebuilt after the first change. Value is a time in ms.
+		 */
+		aggregateTimeout?: number;
+		/**
+		 * Ignore some files from watching
+		 */
+		ignored?: {
+			[k: string]: any;
+		};
+		/**
+		 * Stop watching when stdin stream has ended
+		 */
+		stdin?: boolean;
+		/**
+		 * Enable polling mode for watching
+		 */
+		poll?: boolean | number;
+	};
+}
+export interface Module {
+	/**
+	 * Enable warnings for full dynamic dependencies
+	 */
+	exprContextCritical?: boolean;
+	/**
+	 * Enable recursive directory lookup for full dynamic dependencies
+	 */
+	exprContextRecursive?: boolean;
+	/**
+	 * Sets the default regular expression for full dynamic dependencies
+	 */
+	exprContextRegExp?:
+		| boolean
+		| {
+				[k: string]: any;
+		  };
+	/**
+	 * Set the default request for full dynamic dependencies
+	 */
+	exprContextRequest?: string;
+	/**
+	 * Don't parse files matching. It's matched against the full resolved request.
+	 */
+	noParse?:
+		| {
+				[k: string]: any;
+		  }[]
+		| {
+				[k: string]: any;
+		  }
+		| string[]
+		| string;
+	/**
+	 * An array of rules applied for modules.
+	 */
+	rules?: RuleSetRules;
+	/**
+	 * An array of rules applied by default for modules.
+	 */
+	defaultRules?: RuleSetRules;
+	/**
+	 * Enable warnings when using the require function in a not statically analyse-able way
+	 */
+	unknownContextCritical?: boolean;
+	/**
+	 * Enable recursive directory lookup when using the require function in a not statically analyse-able way
+	 */
+	unknownContextRecursive?: boolean;
+	/**
+	 * Sets the regular expression when using the require function in a not statically analyse-able way
+	 */
+	unknownContextRegExp?:
+		| boolean
+		| {
+				[k: string]: any;
+		  };
+	/**
+	 * Sets the request when using the require function in a not statically analyse-able way
+	 */
+	unknownContextRequest?: string;
+	/**
+	 * Cache the resolving of module requests
+	 */
+	unsafeCache?:
+		| boolean
+		| {
+				[k: string]: any;
+		  };
+	/**
+	 * Enable warnings for partial dynamic dependencies
+	 */
+	wrappedContextCritical?: boolean;
+	/**
+	 * Enable recursive directory lookup for partial dynamic dependencies
+	 */
+	wrappedContextRecursive?: boolean;
+	/**
+	 * Set the inner regular expression for partial dynamic dependencies
+	 */
+	wrappedContextRegExp?: {
+		[k: string]: any;
```

Comment:
This is wrong

Reply:
All `RegExp` are missing

Reply:
Yup this is something we will have to support in the transformation. I've spoke with the maintainer of the json-schema-to-typescript package and there is a way we could support these extra ajv types.